docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -77,7 +77,7 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 
```

Comment:
can you please also update ms.date on line 6 to be 11/28/2018?

Reply:
Fixed.